### ISSUE TITLE
Vat.js File Missing

### ISSUE THREAD

**Issue by aj-agoric** [Agoric/Agoric#9612]
## Describe the bug
User reports they are not finding `vat.js'` anywhere in any of the published packages

fixed by #9607 

## To Reproduce
1. `import * as vv from '@agoric/vow/vat.js';`
2. get `ERR_MODULE_NOT_FOUND`

## Expected behavior
to be able to import `vat.js` from outside agoric-sdk

## Platform Environment
 - what OS are you using? what version of Node.js?
 - is there anything special/unusual about your platform?
 - what version of the Agoric-SDK are you using? (run `git describe --tags --always`)

## Screenshots
If applicable, add screenshots to help explain your problem, especially for UI interactions.

**Comment by dckc** [comment]
fixed by #9607

**Comment by dckc** [comment]
I verified that the released package includes `vat.js`; this [imp.js](https://gist.github.com/dckc/118cb294fe6e286f7529813f761f14d7#file-imp-js)  works now:

```js
import '@endo/init';
import * as it from '@agoric/vow/vat.js';

console.log(Object.keys(it));
```

We get:

```console
$ node imp.js
[ 'defaultPowers', 'heapVowE', 'heapVowTools', 'prepareVowTools' ]
```

package version info:
```
"@agoric/vow@^0.2.0-u16.1":
  version "0.2.0-u16.1"
  resolved "https://registry.yarnpkg.com/@agoric/vow/-/vow-0.2.0-u16.1.tgz#55dafb45acd1c2704be5b624ab4a0e736fe5f109"
  integrity sha512-4MCe+7GsmZwB2i2DU/c2GzNeUwnlbe6FU9dTOOO0baQL2xS0dSszugWGgnWZD4CvTdWc6GhhuzwwDS5w48YsQg==
```

more details are in a gist: [imp.js](https://gist.github.com/dckc/118cb294fe6e286f7529813f761f14d7#file-imp-js) and friends.
